articles/variadic-function-templates.dd

```diff
@@ -132,16 +132,16 @@ $(H3 The D Look Ma No Templates Solution)
 ------
 import core.vararg;
 import core.stdc.stdio;
-import std.format;
-
+//For reference, this function signature is from D1.0
```

Comment:
The point of this example is to show a template-less approach. Removing the FQN from `doFormat` makes sense but mentioning D1 is just confusing IMO. If you really want it mentioned, something like:
"While the standard library favors variadic templates over runtime type information, this approach can be taken whenever variadic templates are not practical, such as when passing a delegate is needed"

Reply:
feedback incorporated, thoughts?